dist/mercator.spec

```diff
@@ -1,23 +1,41 @@
 # golang does not generate debug info
 %define debug_package %{nil}
 
-Name:		mercator
-Version:	1
-Release:	16%{?dist}
-Summary:	Mercator CLI tool
-License:	ASL 2.0
-URL:		https://github.com/fabric8-analytics/%{name}-go
-
-Source0:	%{name}-go.tar.gz
-# prebuild binary
-Source1: %{name}
+Name:       mercator
+Version:    1
+Release:    17%{?dist}
+Summary:    Mercator CLI tool
+License:    ASL 2.0
+URL:        https://github.com/fabric8-analytics/%{name}-go
+
+Source0:    %{name}-go.tar.gz
+
+# pre-built binary & handlers
+Source1:    %{name}
+Source2:    handlers.yml
+Source3:    java
+Source4:    dotnet
+
 # data normalizer source code
-Source2: data_normalizer.py
-Source3: java
-Source4: handlers_template.yml
+Source5:    data_normalizer.py
+
+BuildRequires:  make
+
+# python handler
+%if 0%{?rhel}
+Requires:       python34
+%else
+Requires:       python3
+%endif
+# ruby handler
+Requires:       ruby
+# npm handler
+Requires:       nodejs
+# java handler
+Requires:       java maven
+# dotnet handler
+Requires:       mono-core
```

Comment:
I am not sure whether there is mono-core at all. but the same problem applies here, the installation would fail